webpack/assets/javascripts/react_app/components/TemplateGenerator/TemplateGenerator.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Alert, Button } from 'patternfly-react';
+
+import { sprintf, translate as __ } from '../../../react_app/common/I18n';
+import AlertBody from '../common/Alert/AlertBody';
+
+const pollingMsg = `
+  Report %s is now being generated, the download will start once it's done.
+  You can come to this page later to get the results. The result is available for 24 hours.
+`;
+const doneMsg =
+  'Generation of report %s has completed. Download should start automatically. In case it does not, please use download button below.';
```

Comment:
`Generating of the report %s has been completed. Download should start automatically. In case it does not, please use the download button below.`

Reply:
Yeah sounds better! :+1: